test/common_utils.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+
+
+def generate_unique_name(prefix, timestamp_string) -> str:
+    name = prefix + "_" + str(uuid.uuid4()) + "_" + timestamp_string
+    return name
+
+
+def run_proc(conn_cnx, sql) -> str:
+    with conn_cnx() as cnx:
+        cur = cnx.cursor()
+        result = cur.execute(sql).fetchone()
+        return result
+
+
+def delete_list_of_labels(conn_cnx, sql):
+    print(f"[INFO] SQL in delete function: {sql}")
+
+    with conn_cnx() as cnx:
+        cur = cnx.cursor()
+        for name in cur.execute(sql).fetchall():
+            delete_label_statement = f"call ADMIN.DELETE_LABEL('{name[0]}');"
+            assert "done" in str(
+                run_proc(conn_cnx, delete_label_statement)
+            ), "Stored procedure output does not match expected result!"
+
+
+# Used for validating result correctness
+def validate_row_count(conn_cnx, sql) -> int:
```

Comment:
this method is identical to `run_proc` why have both?

Reply:
They return different data types and I felt it would be bad form to combine them even though they are super simple ...
Plus for readability/usability: it's harder to make a mistake when you have very different function names.
If you feel strongly about it, I will make a change.

Reply:
its just code duplication it doesn't seem useful to me. We are not doing type checks anywhere so the types are only informational to the user,

if you want to be careful about types and more informational I would do something like

``` python
def _get_single_value(conn_ctx: cursor, sql: str):
    with conn() as cnx:
        cur = cnx.cursor()
        result = cur.execute(sql).fetchone()
        return result

def run_proc(conn_ctx: cursor, sql: str) -> str:
    result = _get_single_value(conn_ctx, sql)
    assert isinstance(result, str)
    return result

def row_count(conn_ctx: cursor, sql: str) -> int:
    result = _get_single_value(conn_ctx, sql)
    assert isinstance(result, int)
    return result
```

Reply:
That's elegant! Lemme do it. Thank you.

Reply:
done.